src/resources/grammar/SearchDSL.jison

```diff
@@ -0,0 +1,74 @@
+/* description: Parses end executes mathematical expressions. */
+
+%{
+const {merge_and, merge_or, op_fuzzy, op_attrib, op_exact}
```

Comment:
yes, there is no restriction from JISON, however its syntax uses snake-case in some cases (ex. `@1. last_column`). That's why I used the same approach on the library functions.